mix/ContractCallDataEncoder.cpp

```diff
@@ -196,3 +201,18 @@ QStringList ContractCallDataEncoder::decode(QList<QVariableDeclaration*> const&
 	}
 	return r;
 }
+
+
+bool ContractCallDataEncoder::isString(dev::bytes const& _b, QString& _str)
```

Comment:
name is a bit misleading as this function also does string conversion. asString perhaps?